kbfsmd/root_metadata_v3.go

```diff
@@ -645,11 +645,35 @@ func (md *RootMetadataV3) CheckValidSuccessorForServer(
 	return nil
 }
 
+// IsBackedByTeam returns true if md is for a TLF backed by a team. It could be
+// either a SingleTeam TLF or a private/public TLF backed by an implicit team.
+func (md *RootMetadataV3) IsBackedByTeam(extra ExtraMetadata) bool {
+	if extra != nil {
```

Comment:
If you keep the `extra`argument (see below), I think this check should go below the team check, and it should return an error if `extra != nil`, because that's not supposed to happen.